src/drozer/configuration.py

```diff
@@ -88,7 +88,7 @@ def get_all_values(cls, section):
         cls.__ensure_config()
         
         if cls.__config.has_section(section):
-            return map(lambda k: cls.__config.get(section, k), cls.__config.options(section))
+            return list(map(lambda k: cls.__config.get(section, k), cls.__config.options(section)))
```

Comment:
This should have been a list in the first place. Missed in the py2-to-py3 conversion, and kinda just worked by accident